docs/topics/testing/tools.txt

```diff
@@ -1786,6 +1793,14 @@ your test suite.
 
     Output in case of error can be customized with the ``msg`` argument.
 
+.. method:: TransactionTestCase.assertQuerysetEqual(qs, values, transform=None, ordered=True, msg=None)
+
+    .. deprecated:: 4.2
+
+        This old spelling of ``assertQuerySetEqual()`` is deprecated and will
+        be removed in Django 5.1.
+
+
```

Comment:
I'd add `deprecated:: 4.2` annotation to the `assertQuerysetEqual` instead of documenting a deprecated method.

Reply:
Fixed in https://github.com/django/django/pull/16110/commits/54fed788d0dd9fedc6d5c83a528b3dfff90c0c09.